file-formats/ints/examples/z_aff_example_ints.ints.json

```
@@ -0,0 +1,27 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "ISLM ADT for INTS - Scenario",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "scenarioType": "SAPGENAI",
+    "scenarioTechnology": "sideBySide",
+    "isExtensible": true,
+    "state": "Published",
+    "islmVersion": 1.0
+  },
+  "turnkeyInformation": {
+    "isTurnkeyAutoSwitchOn": true,
+    "turnkeyType": "DEFAULT"
+  },
+  "connectionInformation": {
+    "usageType": "STATELESS",
+    "oauthProfile": "ISLM_SAPGENAI_OAUTH_PRF_HCP"
+  },
+  "modellingContext": {
+    "modellingContextName": "DEFAULT",
+    "modellingContextId": "FD7EE83C8A15E829D272C9A50AB7B0A9",
+    "modellingContextDescription": "Default context"
+  }
```

My team members pointed out that we have some repetition here. Usually, we try to avoid this ("don't repeat yourself")

```suggestion
  "modellingContext": {
    "name": "DEFAULT",
    "id": "FD7EE83C8A15E829D272C9A50AB7B0A9",
    "description": "Default context"
  }
```

We recommend to change it. However, since this is a minor and you anyhow have V2 in mind. You can decide whether you ant to change it with that version.

Same in the `turnkey` structure.

@raghav6676 Just let me know your opinion whether you want to change it in v1 or keep it.

---

Hi @schneidermic0 ,

I vaguely remember that this was a review comment, to keep this full name.
But please let's change it during V2.

Thanks,
Raghav

---

> I vaguely remember that this was a review comment, to keep this full name.

To be honest, this PR has too many comments in the meantime. 

> But please let's change it during V2.

Fine for me 👍